### ISSUE TITLE
Version without 4G dongle

### ISSUE THREAD

**Issue by marcdubois71450** [marcdubois71450/tesla-carplay#23]
We can probably do the same thing without a 4G adapter if we deceive the tesla DNS, this DNS is checked when the tesla connects to the wifi, without this DNS response the tesla displays a DHCP error message.  This would make it possible to do without a 4G subscription and a 4G dongle for this project.  This would make the project accessible to even more people.  I will work on this solution

https://github.com/tesla-android/tesla-android.github.io/issues/36#issuecomment-1257752038

https://github.com/tesla-android/tesla-android-os/tree/main/src/modules/tesla-android/filesystem/etc/pihole

**Comment by cariacou** [comment]
Sounds great!

**Comment by cariacou** [comment]
It seems like another developper was able to bypass the requirement for a 4G connection.
He is using an Android OS, though I am assuming similar method can be used
 
https://tesla-android.gapinski.eu/install-guide

**Comment by marcdubois71450** [comment]
> It seems like another developper was able to bypass the requirement for a 4G connection. He is using an Android OS, though I am assuming similar method can be used
> 
> https://tesla-android.gapinski.eu/install-guide

Yes, that's why I opened this issue, it does this at the DNS level, I already put the links in my first comment.  It cheats this DNS with pihole I would like to do it with bind9.  but I haven't had time to test all of this yet, I use this issue as notes depending on the subject to be treated.

**Comment by cariacou** [comment]
The release notes do confirm Pihole was used, then dropped 

`Connectivity is now handled directly within the Android system, Pi-hole is no longer required. `
